src/js/containers/covid19/homepage/CovidHighlights.jsx

```diff
@@ -65,7 +70,7 @@ export class CovidHighlights extends React.Component {
         this.handleResizeWindow();
         return this.fetchDefCodes()
```

Comment:
```suggestion
        return Promise.all([this.fetchDefCodes(), this.fetchSubmissionMonth() ])
```

Like this ☝️ so that we would fetch the defcodes (pre-requisite to fetching totals) and the `as of` date prior to fetching the data for the totals and highlights.

All these considerations are just for UX.